install/install.sh

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bash
+
+if [ "$#" -ne 1 ]; then
+    echo "Invalid number of parameters"
+		exit 1
+fi
+
+JSONPATH1="{.spec.replicas}"
+JSONPATH2="{.status.readyReplicas}"
+ZkOprName=zoo-op
+ZkName=zoo
+BkOprName=pr
+BkName=br
+PrOprName=foo
+PrName=bar
+
+if [ "$1" == "1" ]; then
```

Comment:
Move install code inside a function and similar for delete. Invoke functions based on argument passed to script. In case of delete, user should specify name of pravega cluster to be deleted.